docs/source/user-guide/model_parameters.md

```diff
@@ -2,49 +2,71 @@
 
 :::{Note}
 The simulation model is a central part of the CTAO Simulation Pipeline. The responsibility for the values and the correctness of the simulation model parameters is with the CTAO Simulation Team.
 :::
 
-## Description of model parameters
+## Description of model parameters through schemas
+
+Model parameter data structures are defined in schema files.
+To ensure consistency and correctness of the model parameters, these schema files are used to validate the model parameter files with `simtools-validate-file-using-schema`.
+Strict versioning of schemas and metaschema is introduced to ensure that the model parameters are always compatible with the schema files.
 
-Schema files define all simulation model parameters are part of [simtools](https://github.com/gammasim/simtools) and can be found in [simtools/schemas/model_parameters](https://github.com/gammasim/simtools/tree/main/simtools/schemas/model_parameters).
+The model parameters are stored in json-style in the [model repository](https://gitlab.cta-observatory.org/cta-science/simulations/simulation-model/simulation-models) and database.
+A typical model parameter file looks like:
+
+```json
+{
+    "schema_version": "0.1.0",
+    "model_parameter_schema_version": "0.1.0",
+    "parameter": "num_gains",
+    "instrument": "LSTN-01",
+    "site": "North",
+    "parameter_version": "1.0.0",
+    "unique_id": null,
+    "value": 2,
+    "unit": null,
+    "type": "int64",
+    "file": false
+}
+```
 
-These files describe the simulation model parameters including (among others fields) name, type, format, units, applicable telescopes, and parameter description.
-They include information about setting and validation activities, data sources, and simulation software.
-The schema files and especially the model parameter descriptions are derived from (and planned to be synchronized with) the [sim_telarray manual](https://www.mpi-hd.mpg.de/hfm/~bernlohr/sim_telarray/).
+Two types of schema definitions need to be distinguished:
 
-The schema files are in human readable YAML format and follow a fixed meta metaschema (see [model_parameter_and_data_schema.metaschema.yml](https://github.com/gammasim/simtools/blob/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml); found in simtools schema directory).
+1. **Meta schema:** The meta schema defines the structure of the model parameter file. It is a multi-document YAML file including several schema versions. The meta schema is used to validate the model parameter files. The `schema_version` field is used to identify the correct meta schema.
+2. **Model parameter schema:** The model parameter schema defines the properties of the model parameter. It is a human-readable YAML file that describes the model parameter in detail. The `model_parameter_schema_version` field is used to identify the correct model parameter schema (optional field; default value is `0.1.0`).
```

Comment:
Why is it optional? Why did we start from 0.1.0?

Reply:
`model_parameter_schema_version` has been forgotten....nothing really to understand. Until now, all model parameter schema files had a single version only, so not surprising we didn't notice.

All model parameter schema files have a version of `0.1.0`. This is how we usually start. Question is now:

- do we want to change all of them to `1.0.0` (or `2.0.0` for` corsika_starting_grammage`), as we are quite sure about the approach
- do we want to add  `model_parameter_schema_version` to all json dicts uploaded to the model parameter repository / database?

I do think we have other schema files with version numbers of `0.1.0` or similar. Do we want to change them all to `1.0.0`?

Reply:
Let's keep the `0.1.0` for now, not a big deal, especially since simtools itself is not yet at `1.0.0` (but we are close). I would add `model_parameter_schema_version` to all json dicts, it seems natural to have it there.